tests/decors.py

```diff
@@ -18,3 +22,25 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def using_car_database(f):
```

Comment:
Tutaj zamiast dodawać i usuwać tabele, lepiej będzie po prostu usunąć wszystkie dane z już istniejącej tabeli.

https://stackoverflow.com/questions/16573802/flask-sqlalchemy-how-to-delete-all-rows-in-a-single-table